esmvaltool/cmorizers/obs/cmorize_obs.py

```diff
@@ -23,7 +23,7 @@
 from esmvalcore._config import read_config_user_file
 from esmvalcore._task import write_ncl_settings
 
-from .utilities import read_cmor_config
+from utilities import read_cmor_config
```

Comment:
please use correct relative import method ie revert this change

Reply:
fixed